opentelemetry-sdk/src/opentelemetry/sdk/logs/severity.py

```diff
@@ -54,3 +54,63 @@ class SeverityNumber(enum.Enum):
     FATAL2 = 22
     FATAL3 = 23
     FATAL4 = 24
+
+
+_STD_TO_OTLP = {
+    10: SeverityNumber.DEBUG,
```

Comment:
How come the interim levels defined like this? (11-19, 21-29, etc.)

Reply:
Curious will numbers in "std" ever be anything other than 10, 20, 30, 40, 50?

Reply:
This follows the log data model specification for mapping source format https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#mapping-of-severitynumber. The step value from one level to another level in logging module is 10 but the it is 5 in log data model, so it is clearly many-to-one mapping for some levels. Higher the value, higher the severity. In general these intermediary levels don't come into play but logging library allows programmers to define the custom log levels if they need, so this addresses that case. For example I would define some `scary` b/w `error` and `critical` level like below 

```python
>>> import logging
>>> SCARY = 45
>>> logging.addLevelName(SCARY, "SCARY")
>>> def scary(self, message, *args, **kws):
...     self._log(SCARY, message, args, **kws)
...
>>> logging.Logger.scary = scary
>>> logger = logging.getLogger(__name__)
>>> logger.scary("Boo")
Boo
```